src/pages/Profile/Component/ProfileChild.jsx

```diff
@@ -1,19 +1,40 @@
 import axios from 'axios'
 import React, { useState } from 'react'
-import { useSelector } from 'react-redux'
+import { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { unfollowUser, followUser } from '../../../store/userSlice'
 import '../Profile.css'
 
 export const ProfileChild = ({userProfile}) => {
-   const [imageFile, setImageFile] = useState([])
-   const {user} = useSelector((store)=>store.authenticate)
+   const {user, token} = useSelector((store)=>store.authenticate)
+   const {users} = useSelector((store)=>store.users)
+   const dispatch = useDispatch()
+   
+   const findUser = (arr)=>{
```

Comment:
I saw this findUser function in Home.jsx too. So it would be better than you separate out common functions into another file so that code is not repeated